pkg/controller/flow/reconcile_test.go

```diff
@@ -59,6 +60,24 @@ func init() {
 var testCases = []controllertesting.TestCase{
 	{
 		Name: "new flow: adds status, action target resolved",
+		InitialState: []runtime.Object{
+			getNewFlow(),
+			getFlowControllerConfigMap(),
+		},
+		ReconcileKey: "test/test-flow",
+		WantResult:   reconcile.Result{},
+		WantPresent: []runtime.Object{
+			getActionTargetResolvedFlow(),
+			func() *channelsv1alpha1.Channel {
```

Comment:
Another place where @dprotaso's builders from https://github.com/knative/serving/pull/1762 would be useful.